docs/modules/operation/pages/alarms/history.adoc

```diff
@@ -6,7 +6,7 @@ The alarm history feature integrates with Elasticsearch to provide long term sto
 
 When enabled, alarms are indexed in Elasticsearch when they are created, deleted, or when any of the "interesting" fields on the alarm are updated (more on this below.)
 
-Alarms are indexed in such a fashion that allows operators to answer the following questions:
+Alarms are indexed in such a fashion that lets operators answer the following questions:
```

Comment:
Ah the problem with global search and replace! Have updated to the following:
Alarms are indexed so that operators can answer the following questions: